src/components/booksSlice.js

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [
+  { id: 1, title: 'Book 1', author: 'Author 1' },
+  { id: 2, title: 'Book 2', author: 'Author 2' },
+  { id: 3, title: 'Book 3', author: 'Author 3' },
+];
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.filter((book) => book.id !== action.payload);
+    },
+  },
+});
+
+export const { addBook, removeBook } = booksSlice.actions;
+
+export default booksSlice.reducer;
```

Comment:
- [ ] Please remove this redux slice because you already created one in `src/redux/books/books.js` and try to achieve the DRY role and make your code clean.